client/me/help/main.jsx

```diff
@@ -116,88 +117,101 @@ class Help extends React.PureComponent {
-					target="__blank"
-				>
-					<div className="help__support-link-section">
-						<h2 className="help__support-link-title">
-							{ this.props.translate( 'Quick help video tutorials' ) }
-						</h2>
-						<p className="help__support-link-content">
-							{ this.props.translate(
-								'These short videos will demonstrate some of our most popular features.'
-							) }
-						</p>
-					</div>
-				</CompactCard>
-				<CompactCard
-					className="help__support-link"
-					href="https://dailypost.wordpress.com/blogging-university/"
-					target="__blank"
-				>
-					<div className="help__support-link-section">
-						<h2 className="help__support-link-title">
-							{ this.props.translate( 'Self-guided email courses for site owners and bloggers' ) }
-						</h2>
-						<p className="help__support-link-content">
-							{ this.props.translate(
-								'Pick from our ever-growing list of free email courses to improve your knowledge.'
-							) }
-						</p>
-					</div>
-				</CompactCard>
-				<CompactCard
-					className="help__support-link"
-					href="https://learn.wordpress.com"
-					target="__blank"
-				>
-					<div className="help__support-link-section">
-						<h2 className="help__support-link-title">
-							{ this.props.translate( 'Self-guided online tutorial' ) }
-						</h2>
-						<p className="help__support-link-content">
-							{ this.props.translate(
-								'A step-by-step guide to getting familiar with the platform.'
-							) }
-						</p>
-					</div>
-				</CompactCard>
-				<CompactCard
-					className="help__support-link help__support-link-contact"
-					href="/help/contact/"
-				>
-					<div className="help__support-link-section">
-						<h2 className="help__support-link-title">{ this.props.translate( 'Get in touch' ) }</h2>
-						<p className="help__support-link-content">
+			<>
+				<h2 className="help__section-title">{ this.props.translate( 'More Resources' ) }</h2>
+				<div className="help__support-links">
+					<CompactCard
+						className="help__support-link"
+						href={ localizeUrl( 'https://wordpress.com/support/' ) }
+						target="__blank"
+					>
+						<Gridicon icon="book" size={ 36 } />
+						<div className="help__support-link-section">
+							<h2 className="help__support-link-title">
+								{ this.props.translate( 'All articles' ) }
+							</h2>
+							<p className="help__support-link-content">
+								{ this.props.translate(
+									'Looking to learn more about a feature? Our docs have all the details.'
+								) }
+							</p>
+						</div>
+					</CompactCard>
+					<CompactCard
+						className="help__support-link"
+						href={ localizeUrl( 'https://wordpress.com/support/video-tutorials/' ) }
+						target="__blank"
+					>
+						<Gridicon icon="video" size={ 36 } />
+						<div className="help__support-link-section">
+							<h2 className="help__support-link-title">
+								{ this.props.translate( 'Video tutorials' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'Quick help video tutorials' )` *ES Score: 8*